kitsune/products/tests/test_templates.py

```diff
@@ -158,3 +159,52 @@ def test_subtopics(self):
         eq_(200, r.status_code)
         pqdoc = pq(r.content)
         eq_(1, len(pqdoc('li.subtopic')))
+
+    def test_community_support_shown_for_default_language(self):
+        """Verify the get community support is shown in the topics for en-US."""
+        p = ProductFactory()
+        d = DocumentFactory(slug='get-community-support', locale=settings.LANGUAGE_CODE)
+        rev = ApprovedRevisionFactory(document=d)
+        d.current_revision = rev
+        d.save()
+
+        self.refresh()
```

Comment:
It's in `kitsune/search/tests/__init__.py` - `ElasticTestCase.refresh()`. I assume it's needed to properly show everything needed on the landing page, as the above tests use it too.